### PR TITLE
Gough 7-8

### PR DESCRIPTION
**Pull request by maijapapaya**
@jaredh159 @Henderjay

### CODE REVIEW

**Comment by maijapapaya** [comment]
So in chapter 8 there are some more charts.

File: history-of-the-quakers-v4/original/13-book-7-chapter-7.adoc [hunk 1]
```diff
@@ -205,6 +222,9 @@ A petition was accordingly drawn up in the name of the prisoner,
 setting forth the matter of fact, and consequent proceedings thereupon,
 and further alleged,
 
+[.embedded-content-document.letter]
+--
+
 that she had no intention of disturbing the public worship;
```
Comment (maijapapaya) [inline history-of-the-quakers-v4/original/13-book-7-chapter-7.adoc:228]: Should I use ellipses here again?

File: history-of-the-quakers-v4/original/14-book-7-chapter-8.adoc [hunk 2]
```diff
@@ -32,9 +37,11 @@ In the mean time it was thought expedient to prepare an epistle by way of introd
 as to reasons for presenting the apology, which being drawn up and approved,
 was printed in English and Latin, and is as followeth.
 
-To the Ambassadors of the Christian Princes and States,
-met to conclude a general Peace at Aix-la-Chapelle,
-the People called Quakers wish Health and Happiness.
+[.embedded-content-document.epistle]
+--
```
Comment (maijapapaya) [inline history-of-the-quakers-v4/original/14-book-7-chapter-8.adoc:41]: This is called an epistle, but is it?

File: history-of-the-quakers-v4/original/13-book-7-chapter-7.adoc [hunk 3]
```diff
@@ -85,14 +96,20 @@ tranquility of Europe on a lasting foundation,
...
 
+--
+
 Which address the king was pleased to receive very graciously,
 and to which the following answer was returned.
 
+[quote]
+____
 I thank you for this address of congratulation,
 the just concern you have shown there in for hie and my family is very agreeable to me;
```
Comment (jaredh159) [inline history-of-the-quakers-v4/original/13-book-7-chapter-7.adoc:110]: Can you fix the `hie` in this sentence while you're at it?  Should be `me`.

File: history-of-the-quakers-v4/original/13-book-7-chapter-7.adoc (hunk 1 above)
Comment (jaredh159) [reply history-of-the-quakers-v4/original/13-book-7-chapter-7.adoc:228]: yeah, i think so. `&hellip;that she had <etc>`

File: history-of-the-quakers-v4/original/14-book-7-chapter-8.adoc [hunk 4]
```diff
@@ -313,32 +330,40 @@ we recommend what follows to your serious consideration, viz.
 A brief Account of the Origin of the Names of some Months of the Year,
```
Comment (jaredh159) [inline history-of-the-quakers-v4/original/14-book-7-chapter-8.adoc:330]: This seems like it needs to be a  blurb-style subheading, i think.

File: history-of-the-quakers-v4/original/14-book-7-chapter-8.adoc [hunk 5]
```diff
@@ -1724,8 +1777,10 @@ O grave where is thy victory?
 but thanks be to God who giveth us the victory.`" Many friends
 from London and elsewhere went to visit him,
 which he took very kindly,
-and to one of them he expressed him self in the following manner,
+and to one of them he expressed himself in the following manner,
 
+[quote]
+____
```
Comment (jaredh159) [inline history-of-the-quakers-v4/original/14-book-7-chapter-8.adoc:1783]: I think let's switch this one to an embedded testimony.

Comment (maijapapaya) [reply history-of-the-quakers-v4/original/14-book-7-chapter-8.adoc:1783]: I always forget about that testimony when it's someone speaking!!

**Comment by jaredh159** [comment]
Thanks, merging.

**Pull request merged**